app/uk/gov/hmrc/merchandiseinbaggagefrontend/config/AppConfig.scala

```diff
@@ -33,12 +33,24 @@ class AppConfig() extends MongoConfiguration {
   val contactUrl = s"$contactHost/contact/contact-hmrc-unauthenticated?service=$serviceIdentifier"
 }
 
+object AppConfigSource {
+  val configSource: String => ConfigSource = ConfigSource.default.at
+}
+
 trait MongoConfiguration {
   lazy val mongoConf: MongoConf = configSource("mongodb").loadOrThrow[MongoConf]
 }
 
 final case class MongoConf(uri: String, host: String = "localhost", port: Int = 27017, collectionName: String = "declaration")
 
-object AppConfigSource {
-  val configSource: String => ConfigSource = ConfigSource.default.at
-}
\ No newline at end of file
+trait MibConfiguration {
```

Comment:
Please could you use Guice for this consistent with the other config - e.g. see https://github.com/hmrc/merchandise-in-baggage-frontend/blob/f067c39cf8b5d590f3f0e8387e27e95b046129bf/app/uk/gov/hmrc/merchandiseinbaggagefrontend/config/MerchandiseInBaggageFrontendConfigModule.scala#L31

Reply:
looks like you did already use Guice, I think this should be removed.